source/documentors/quickstarts/first_documentation_pr.rst

```diff
@@ -63,15 +63,16 @@ repo.
 
```

Comment:
I'd add a sentence here to the effect of

> A "fork" is essentially your own copy of the repository at a single point in time

Reply:
good idea, I dropped the "at a single point in time" bit since that raises more questions.